package.json

```diff
@@ -39,57 +39,52 @@
   "author": "Postman Labs <help@getpostman.com> (=)",
   "license": "Apache-2.0",
   "dependencies": {
-    "@postman/csv-parse": "4.0.2",
     "async": "3.2.0",
-    "chardet": "0.8.0",
+    "chardet": "1.0.0",
     "cli-progress": "3.6.1",
-    "cli-table3": "0.5.1",
+    "cli-table3": "0.6.0",
     "colors": "1.4.0",
-    "commander": "4.1.1",
+    "commander": "5.0.0",
+    "csv-parse": "4.8.8",
     "eventemitter3": "4.0.0",
     "filesize": "6.1.0",
     "lodash": "4.17.15",
-    "mkdirp": "0.5.1",
+    "mkdirp": "1.0.4",
     "postman-collection": "3.6.0",
     "postman-collection-transformer": "3.3.1",
     "postman-request": "2.88.1-postman.21",
     "postman-runtime": "7.24.1",
-    "pretty-ms": "5.1.0",
-    "semver": "7.1.3",
+    "pretty-ms": "6.0.1",
+    "semver": "7.2.1",
     "serialised-error": "1.1.3",
     "word-wrap": "1.2.3",
-    "xmlbuilder": "14.0.0"
+    "xmlbuilder": "15.1.0"
   },
   "devDependencies": {
     "chai": "4.2.0",
     "dockerfile_lint": "0.3.3",
     "editorconfig": "0.15.3",
-    "eslint": "5.16.0",
-    "eslint-plugin-jsdoc": "8.7.0",
+    "eslint": "6.8.0",
+    "eslint-plugin-jsdoc": "22.1.0",
     "eslint-plugin-lodash": "6.0.0",
     "eslint-plugin-mocha": "6.3.0",
     "eslint-plugin-security": "1.4.0",
     "js-yaml": "3.13.1",
-    "jsdoc": "3.6.3",
+    "jsdoc": "3.6.4",
     "jsdoc-to-markdown": "5.0.3",
     "mocha": "7.1.1",
-    "nock": "10.0.6",
-    "nyc": "14.1.1",
+    "nock": "12.0.3",
+    "nyc": "15.0.1",
     "packity": "0.3.2",
     "parse-gitignore": "1.0.1",
     "postman-jsdoc-theme": "0.0.3",
     "recursive-readdir": "2.2.2",
     "server-destroy": "1.0.1",
     "shelljs": "0.8.3",
-    "sinon": "7.5.0",
+    "sinon": "9.0.1",
     "xml2js": "0.4.23"
   },
   "engines": {
-    "node": ">=6"
-  },
-  "greenkeeper": {
-    "ignore": [
-      "eslint-plugin-jsdoc"
-    ]
+    "node": ">=8"
```

Comment:
As mentioned in the MIGRATION guide:
> Newman v5.0  drops support for outdated Node.js versions (still works on Node.js v8, but only v10 and above are officially supported).

This is kept to `>=8` temporarily because of two reasons:
1. Still works on Node v8
2. We are dropping support for both v6 and v8 in one shot

Newman CLI has a node engine check and keeping this `>=10` means on `v8` CLI will not work but library usage will work perfectly fine.

Maybe we can just keep it `10` as well 💁🏻‍♂️

Reply:
Why don't we keep the version 8 in `.travis` as well while officially dropping the support? Then if travis ever breaks for node 8, we can remove it from `.travis` and `package.json` at the same time.

Or we remove node 8 from everywhere to avoid confusion.

Thoughts?

Reply:
Yes, changed to `>=10`.